widgets/multiple-choice-list/index.js

```diff
@@ -1,24 +1,60 @@
 var React = require('react');
+var cx = require('classnames');
 
 var utils = require('../../lib/widgetUtils.js');
-var defaultTemplate = require('./template.html');
 
-function multipleChoiceList({container = null, facetName = null, template = defaultTemplate}) {
+var defaultTemplate = `<label>
+  <input type="checkbox" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{name}} <span>{{count}}</span>
+</label>`;
+
```

Comment:
The thing is that the more webpack-loaders we use, the more it will be hard for people to use us because loaders from libraries are not automatically loaded when you require(library).

I am not sure ATM how to specify loaders in webpack.config.js that will be loaded when requiring the library. I think this is not feasible.

The other solution would be to expose the builded library as the main entry point (and push it in npm like many libraries are doing).

So that browserify or webpack users will have the right build. That would work.

Also I thought that for a simple 3 lines template, using multiline string was ok.